### PR TITLE
How to set a virtual environment, updated version 

### PR DESCRIPTION
**Pull request by san89**

### CODE REVIEW

**Comment by Dimkoim** [comment]
Hello Santiago and well done! I have some remarks at your commit. Please have a look.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,18 +27,25 @@ Understanding certain steps in the process assumes a basic grasp of certain impo
...
-1. Make sure you have a relatively clean <a href="https://www.anaconda.com/download/#windows">Anaconda installation</a>.
-2. Create a new [conda virtual environment](https://conda.io/docs/user-guide/tasks/manage-environments.html): `conda create --name kaggle_env --file requirements.txt;` 
-3. Activate the newly created environment: `source activate kaggle_env`. This step should be repeated everytime you start working on the project.
-4. In case you are using an IDE like Pycharm or Spyder, make sure it uses your conda environment as the project interpreter.
-	- Your environment will be by default located at: `<Anaconda Home>/envs/kaggle_env/python.exe`
-	- Default location for `<Anaconda Home>` on Windows is `C:\Users\<username>\AppData\Local\Continuum\anaconda3\`
-5. In case your changes required the installation of extra packages (for example via `conda install <package>` remember to update the `requirements.txt` file: `conda list --explicit > requirements.txt`. 
-   This way others can install them with `conda install --file requirements.txt`
-6. You can leave the virtual enviroment at any time using `source deactivate`
+1. Install `virtualenv` using `pip install virtualenv`
```
Comment (Dimkoim) [inline CONTRIBUTING.md]: The Window users should open a cmd window.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,18 +27,25 @@ Understanding certain steps in the process assumes a basic grasp of certain impo
...
+2. Make sure that you have a relatively clean `base` environment. You can check this using `pip freeze`
```
Comment (Dimkoim) [inline CONTRIBUTING.md]: What is a  relatively `base` environment?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,18 +27,25 @@ Understanding certain steps in the process assumes a basic grasp of certain impo
...
+3. Go to the folder where the `jads_kaggle` repository is located using `cd .../jads_kaggle`
+4. Create the virtual environment using `virtualenv kaggle_env`. Note that this new environment will copy all the libraries that exist in the `base` environment
+5. Now your environment is ready to be used. You can activate it using `activate kaggle_env` or deactivate it just using `deactivate` command.
+
```
Comment (Dimkoim) [inline CONTRIBUTING.md]: I open the Anaconda Navigator and I cannot see the new enviroment. How can I add it?

File: CONTRIBUTING.md (hunk 1 above)
Comment (san89) [reply CONTRIBUTING.md]: I agree. I am assuming that all of us will use windows. I understand that a project should be developed using only one operating system. Is that right?

File: CONTRIBUTING.md (hunk 2 above)
Comment (san89) [reply CONTRIBUTING.md]: a base environment is the one that we have by default in our system I think. Should I provide an explanation?

File: CONTRIBUTING.md (hunk 3 above)
Comment (san89) [reply CONTRIBUTING.md]: I have exactly the same question. I have done some research on that but I haven't found a robust solution. Do you have some suggestions Mimis?

File: CONTRIBUTING.md (hunk 2 above)
Comment (Dimkoim) [reply CONTRIBUTING.md]: I don't get what clean means.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Dimkoim) [reply CONTRIBUTING.md]: Manos is working in Ubuntu.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Dimkoim) [reply CONTRIBUTING.md]: I am working on that right now.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -27,18 +27,26 @@ Understanding certain steps in the process assumes a basic grasp of certain impo
...
+3. Install `pip install virtualenvwrapper-win`
+4. Create the virtual environment using `mkvirtualenv kaggle_env`
+5. Go to the folder where the `jads_kaggle` repository is located using `cd .../GitHub/jads_kaggle`
+6. In this folder set `kaggle_env` using the command `setprojectdir .`. Now, every time that you activate `kaggle_env` (using the command `workon kaggle_env`) the path `cd .../jads_kaggle` will be automatically set
+7. To activate the virtual environment in Anaconda, you have to use `Anaconda Prompt` and type `workon kaggle_env`. To activate the virtual environment in PyCharm type `workon kaggle_env` on the PyCharm terminal
+8. You can return to the `base environment` anytime using the command `deactivate`
+
+# Deleting a Conda Virtual Environment
+
+In order to delete a conda virtual enviromen you can:
```
Comment (Dimkoim) [inline CONTRIBUTING.md]: enviroment

**Pull request merged**